meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_moderate_form.html

```diff
@@ -32,6 +32,9 @@ <h1>{% trans 'Moderate proposal' %}</h1>
             {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.statement.statement %}
             {% include 'meinberlin_contrib/includes/form_checkbox_field.html' with field=forms.moderateable.is_archived %}
             {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.remark.remark %}
+            {% if forms.moderateable.show_tasks %}
+                {% include 'meinberlin_contrib/includes/form_field.html' with field=forms.moderateable.completed_tasks %}
+            {% endif %}
```

Comment:
this should be before the remark

Reply:
Oh no, that makes the top divider more complicated, though. :(

Reply:
And here it's the same order: https://app.zeplin.io/project/5b051eef7aaa417d2227c55c/screen/633edb11a565dd9ca59cbde0 :relieved:

Reply:
@fuzzylogic2000 oooo I was looking at this one and as it's also the order it's displayed in it does make more sense https://app.zeplin.io/project/5b051eef7aaa417d2227c55c/screen/6375fd63bc5e2c6da744e5e6

Reply:
Yes, it does make more sense! Will find out which one we should take.